### DISCUSSION TITLE
How to make the blog my home page?

### DISCUSSION THREAD

**Discussion by ghost** [datenstrom/community#936] [Ask a question] [answered]
I have started using Datenstrom Yellow, and it is so brilliant.
I have just taken up the Blog site function - but the domain I have it on is intended for writing blog entries, so I would like to land directly on the blog page. Can you help me point it over?

**Comment by PetersOtto** [comment, marked as answer]
Simply delete the `1-home ` folder.

**Reply by ghost** [reply]
Aha! 
Not surprised to hear an elegant solution with good design. 
Thank you so much.

**Reply by ghost** [reply]
Sorry to say this did not resolve the issue. 
If I simply remove the Home folder, DSY will still look for home. I renamed to 2-Home and 1-Blog - it does change the sequence of menu buttons, but the site still opens to the Home page when I go to / .

**Reply by PetersOtto** [reply]
Deleting the `1-home` folder should work. Have you tried deleting the browser cache?

**Reply by ghost** [reply]
It turns out it did work when I dropped in a fresh installation and removed it. Now it plays right.

**Reply by kolewu** [reply]
It does not work to just have the blog directory as the first one in `content/`, if the blog is setup to use subdirectories for `BlogNewLocation`, because the startpage is not allowed to have subfolders. But the solution from  #1007 does work then.

**Reply by annaesvensson** [reply]
What are your `BlogStartLocation` and `BlogNewLocation`? Have a look at [#239](https://github.com/datenstrom/community/discussions/239#discussioncomment-244744) and #881, it's a bit tricky with subfolders.

**Reply by kolewu** [reply]
It's as in example 2:
```
BlogStartLocation: /blog/
BlogNewLocation: /blog/@year/@title
```

**Reply by annaesvensson** [reply]
Ah thanks, now I understand what's happening. For a blog without subfolders it's super easy, simply delete the home folder. For a blog with subfolders, you first have to change the blog settings and then adjust the content structure. As you said, it does not work otherwise. Have a look at [#239](https://github.com/datenstrom/community/discussions/239#discussioncomment-244744), example 5 should explain the setup you want.

**Reply by kolewu** [reply]
Thank you, but it is not and it seems like a similar monk than for @pftnhr ;-) Having the homepage as startpage for the blog like in the answer from @markseuffert in #881 is perfect.

**Reply by markseuffert** [reply]
How lucky that we support the Vulcan concept of infinite diversity in infinite combination. 🖖
